raiden/network/transport/matrix/rtc/web_rtc.py

```diff
@@ -330,17 +329,19 @@ def send_message(self, message: str) -> None:
         self.schedule_task(self._send_message(message))
 
     async def _close(self) -> None:
+        if self._channel is None:
```

Comment:
I think this could lead to something wrong... 
The callee only receives the channel object after the channel is established. During the signalling process the object is still None. 

Nonetheless, if teardown is started during signalling process, the proper teardown with eerconnection.close()` etc should be executed